tests/system/small/ml/test_llm.py

```diff
@@ -17,11 +17,29 @@
 from bigframes.ml import llm
 
 
-def test_create_text_generator_model(palm2_text_generator_model):
+def test_create_text_generator_model(palm2_text_generator_model, dataset_id):
     # Model creation doesn't return error
     assert palm2_text_generator_model is not None
     assert palm2_text_generator_model._bqml_model is not None
 
+    # save, load to ensure configuration was kept
+    reloaded_model = palm2_text_generator_model.to_gbq(
+        f"{dataset_id}.temp_text_model", replace=True
+    )
+    assert f"{dataset_id}.temp_text_model" in reloaded_model._bqml_model.model_name
```

Comment:
why not `assert f"{dataset_id}.temp_text_model" == reloaded_model._bqml_model.model_name`?

Reply:
it contains project name I guess?

Reply:
Hm, in that case more precise would be to `assert reloaded_model._bqml_model.model_name.endswith(f"{dataset_id}.temp_text_model")`